src/components/button-panel.js

```diff
@@ -1,39 +1,43 @@
-/* eslint-disable jsx-quotes */
+/* eslint-disable react/prop-types */
+
```

Comment:
Instead of disabling the linter, can you add clickHandler to the PropTypes instead? Seems it's a function, so consider adding a function type for it. see https://reactjs.org/docs/typechecking-with-proptypes.html for example and more explanation.

Reply:
Okay, thanks